src/test/java/software/amazon/payloadoffloading/S3DaoTest.java

```diff
@@ -0,0 +1,75 @@
+package software.amazon.payloadoffloading;
+
+import software.amazon.awssdk.core.sync.RequestBody;
+import software.amazon.awssdk.services.s3.S3Client;
+import software.amazon.awssdk.services.s3.model.ObjectCannedACL;
+import junitparams.JUnitParamsRunner;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.ArgumentCaptor;
+import software.amazon.awssdk.services.s3.model.PutObjectRequest;
+import software.amazon.awssdk.services.s3.model.ServerSideEncryption;
+
+import static org.junit.Assert.*;
```

Comment:
This is very small, but in the team we don't use import *, I saw I missed that on my PR for the same change :)

Reply:
fixed